adoc/chapters/programming_interface.adoc

```diff
@@ -5186,718 +5173,637 @@ If no template argument is provided, then the default allocator for the SYCL
-via the SYCL [code]#device# class info query
-[code]#info::device::mem_base_addr_align#.  Violating this requirement causes
-the implementation to throw an [code]#exception# with the [code]#errc::invalid#
-error code from the [code]#accessor# constructor (if the accessor is not a
-placeholder) or from [code]#handler::require()# (if the accessor is a
-placeholder).  If the accessor is bound to a <<command-group>> with a secondary
-queue, the sub-buffer's alignment must be compatible with both the primary
-queue's device and the secondary queue's device, otherwise this exception is
-thrown.
+* An [code]#exception# with the [code]#errc::invalid# error code if
+[code]#subRange# corresponds to a a non-contiguous region of [code]#b#.
 
-Must throw an [code]#exception# with the [code]#errc::invalid# error code if
-[code]#b# is a sub-buffer.
+* An [code]#exception# with the [code]#errc::invalid# error code if the sum of
+[code]#baseIndex# and [code]#subRange# in any dimension exceeds the parent
+buffer [code]#b# size [code]#bufferRange# in that dimension.
 
-|====
+'''
 
 
+[[sec:buffer-member-funcs]]
+==== Member functions
 
-[[table.members.buffer]]
-.Member functions for the [code]#buffer# class
-[width="100%",options="header",separator="@",cols="65%,35%"]
-|====
-@ Member function @ Description
-a@
-[source]
+.[apititle]#buffer::get_range#
+[source,role=synopsis,id=api:buffer-get-range]
 ----
-range<Dimensions> get_range() const
+range<Dimensions> get_range() const;
 ----
-   a@ Return a range object representing the
-      size of the buffer in terms of number
-      of elements in each dimension as passed
-      to the constructor.
+_Returns:_ A range object representing the size of the buffer in terms of number
+of elements in each dimension as passed to the constructor.
 
-a@
-[source]
-----
-size_t size() const noexcept
-----
-   a@ Returns the total number of elements in the buffer.
-      Equal to [code]#+get_range()[0] * ... * get_range()[Dimensions-1]+#.
-a@
-[source]
-----
-size_t get_count() const
-----
-   a@ Returns the same value as [code]#size()#. Deprecated.
-a@
-[source]
-----
-size_t byte_size() const noexcept
-----
-   a@ Returns the size of the buffer storage in bytes.
-      Equal to [code]#size()*sizeof(T)#.
-a@
-[source]
+'''
+
+.[apititle]#buffer::byte_size#
+[source,role=synopsis,id=api:buffer-byte-size]
 ----
-size_t get_size() const
+size_t byte_size() const noexcept;
 ----
-   a@ Returns the same value as [code]#byte_size()#. Deprecated.
-a@
-[source]
+_Returns:_ The size of the buffer storage in bytes.
+Equal to [code]#size()*sizeof(T)#.
+
+'''
+
+.[apititle]#buffer::size#
+[source,role=synopsis,id=api:buffer-size]
 ----
-AllocatorT get_allocator() const
+size_t size() const noexcept;
 ----
-   a@ Returns the allocator provided to the buffer.
+mode and target in the command group buffer.
+The value of target can be [code]#target::device#,
+[code]#target::constant_buffer# or [code]#target::host_task#.
```

Comment:
This looks broken, but I can't tell if it's just GitHub being weird.

Reply:
Ah, good catch. This looks like a copy-paste error. It should be better now.